activiti-services/activiti-services-query/activiti-services-query-rest/src/test/java/org/activiti/services/query/events/handlers/TaskCreatedEventHandlerIT.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright 2017 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.services.query.events.handlers;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Date;
+import java.util.Optional;
+
+import org.activiti.engine.ActivitiException;
+import org.activiti.services.query.app.repository.ProcessInstanceRepository;
+import org.activiti.services.query.app.repository.TaskRepository;
+import org.activiti.services.query.events.TaskCreatedEvent;
+import org.activiti.services.query.model.ProcessInstance;
+import org.activiti.services.query.model.Task;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.SpringBootConfiguration;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.context.annotation.Import;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.junit4.SpringRunner;
+
+/**
+ * TaskCreatedEventHandler JPA Repository Integration Tests
+ * 
+ * @author Igor Dianov

```

Comment:
As @Salaboy mentioned we are not using the `@author` annotation in the project.

Reply:
Fixed in https://github.com/Activiti/Activiti/pull/1464/commits/d8722dda92f703643b94487ffd962d93586e2eb1